lib/workers/global/index.ts

```diff
@@ -44,13 +46,32 @@ function haveReachedLimits(): boolean {
   return false;
 }
 
+/* istanbul ignore next */
+function checkEnv(): void {
+  const range = pkg['engines-next'].node;
+  if (process.release?.name !== 'node') {
+    logger.warn(
+      { release: process.release },
+      'Unsuported node environment detected.'
+    );
+  } else if (!satisfies(process.versions?.node, range)) {
+    logger.warn(
+      { versions: process.versions, range },
+      'Unsuported node environment detected. Please update node version.'
+    );
+  }
```

Comment:
- else if no process.versions.node then warn about node version cannot be detected (or INFO?)
- else if version doesn't satisfy engines.node then log an error
- else if engines-next.node exists and version doesn't satisfy then log a warning [there will often be times when we are not in the process of deprecating anything?]

Reply:
![image](https://user-images.githubusercontent.com/1798109/109817970-cc233300-7c32-11eb-8d62-0757fc3729e5.png)
so warning is logged it version is missing

So you want some different log levels and messages? 🙃

Reply:
I think if we can't detect the version then it should be a different log message than if we detect it and it's wrong